types/can-autoplay/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for can-autoplay 3.0
+// Project: https://github.com/video-dev/can-autoplay
+// Definitions by: Viacheslav Borodulin <https://github.com/vborodulin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Options {
+    inline?: boolean;
+    muted?: boolean;
+    timeout?: number;
+}
+
+interface CheckResponse {
+    result: boolean;
+    error: Error;
+}
+
+type CheckMethod = (options?: Options) => Promise<CheckResponse>;
+
+declare const canAutoPlay: {
+    audio: CheckMethod;
+    video: CheckMethod;
+};
+
+export = canAutoPlay;
```

Comment:
I recommend the following for clarity:
```suggestion
export function audio(options?: Options): Promise<CheckResponse>;
export function video(options?: Options): Promise<CheckResponse>;
```
and then you can remove the `CheckMethod` type.

Reply:
it makes sense, thx)